### PR TITLE
Doc 411 debug for cluster creation getting stuck

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR introduces a troubleshooting article related to cluster deployments.  A new page is created in the troubleshooting section dedicated to cluster deployment scenarios.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/992#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (karl-cardenas-coding) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:4]:
```suggestion
metaDescription: "Troubleshooting steps for errors during a cluster deployments."
```

**Review by ritawatson — CHANGES_REQUESTED** [review]
In this review:
-  I offered some wording suggestions. 
- Suggest switching steps 4 and 5 under "Debugging Steps" so the user can fix a non-working kubelet service before going on to export the kubeconfig, connect API, etc. 
- Suggest breaking up the second Scenario into two bullets with related paragraphs (I'm new to Markdown, so I hope I've indicated this correctly). The two bullet points are:
    - A bootstrap error might have occurred.
    - The VM may not have an outgoing connection.
- In the InfoBox, I suggest making the command monospace indicated with `` `` (which I hope is correct Markdown. (looked it up)

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:3]:
```suggestion
metaTitle: "Troubleshooting steps for errors during a cluster deployment"
```

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:18]:
```suggestion
## Scenario - Instances Continuously Delete Every 30 Minutes
```

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:51]: I suggest moving non-working kubelet steps ahead of correctly working kubelet steps. 
The step progression would be:
3. Verify the kubelet service is operational.
4. If the kubelet service does not work as expected, ...
5. If the Kubelet service works as expected, do the following: 
6. Check stdout ...

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:82]:
```suggestion
4. If the kubelet service does not work as expected, do the following. If the service operates correctly, you can skip this step. 
```

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:69]:
```suggestion
		- For clusters using Gateway, verify the connection between the Installer and Gateway instance:
```

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:101]: Indent lines 101, 103, and 105 so they are part of the first bullet.

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/04-cluster-deployment.md:109]: Indent this paragraph so it's part of the second bullet.

**Review by ritawatson — CHANGES_REQUESTED** [review]
Just a couple more suggestions and minor capitalization fixes: 
- Steps for working kubelet and non-working kubelet should be switched. Non-working kubelet should come first to give user info on how to fix it before proceeding. If kubelet works, user can skip the step. 
- Adjust paragraph indents in second scenario (gateway installer) so text aligns with bullet text.

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (karl-cardenas-coding) [reply content/docs/14-troubleshooting/04-cluster-deployment.md:82]: @ritawatson this is the name of the agent, so it should be capitalized.

File: content/docs/14-troubleshooting/04-cluster-deployment.md
Comment (karl-cardenas-coding) [reply content/docs/14-troubleshooting/04-cluster-deployment.md:51]: ✅

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/992#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/992#pullrequestreview-1197954361" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
